Podfile.lock

```diff
@@ -20,7 +20,7 @@ PODS:
     - GoogleUtilities/Environment (~> 5.2)
     - GTMSessionFetcher/Core (~> 1.1)
   - FirebaseAuthInterop (1.0.0)
-  - FirebaseCore (5.1.4):
+  - FirebaseCore (5.1.0):
```

Comment:
The version of FirebaseCore is downgraded.

Reply:
@kawoou I declare FirebaseCore using '5.1.4' explicitlycitly 😄